cypress/integration/pages/articles/index.js

```diff
@@ -1,6 +1,31 @@
 import runTestsForPage from '../../../support/helpers/runTestsForPage';
-import tests from './tests';
-import testsForAMPOnly from './testsForAMPOnly';
-import testsForCanonicalOnly from './testsForCanonicalOnly';
+import {
+  testsThatAlwaysRun,
+  testsThatFollowSmokeTestConfig,
+  testsThatNeverRunDuringSmokeTesting,
+} from './tests';
+import {
+  testsThatAlwaysRunForAMPOnly,
+  testsThatFollowSmokeTestConfigForAMPOnly,
+  testsThatNeverRunDuringSmokeTestingForAMPOnly,
+} from './testsForAMPOnly';
+import {
+  testsThatAlwaysRunForCanonicalOnly,
+  testsThatFollowSmokeTestConfigForCanonicalOnly,
+  testsThatNeverRunDuringSmokeTestingForCanonicalOnly,
+} from './testsForCanonicalOnly';
 
-runTestsForPage('articles', tests, testsForCanonicalOnly, testsForAMPOnly);
+const testsForPage = {
+  pageType: 'articles',
+  testsThatAlwaysRun,
```

Comment:
Thanks for renaming. This is much clearer than the previous naming convention in this PR.